validator/accounts/account.go

```diff
@@ -39,13 +40,10 @@ func VerifyAccountNotExists(directory string, password string) error {
 // generates a BLS private and public key, and then logs the serialized deposit input hex string
 // to be used in an ETH1.0 transaction by the validator.
 func NewValidatorAccount(directory string, password string) error {
-	// First, if the keystore already exists, throws an error as there can only be
-	// one keystore per validator client.
-	if err := VerifyAccountNotExists(directory, password); err != nil {
-		return fmt.Errorf("validator account exists: %v", err)
-	}
-	shardWithdrawalKeyFile := directory + params.BeaconConfig().WithdrawalPrivkeyFileName
-	validatorKeyFile := directory + params.BeaconConfig().ValidatorPrivkeyFileName
+	t := time.Now()
+
+	shardWithdrawalKeyFile := directory + params.BeaconConfig().WithdrawalPrivkeyFileName + t.Format("20060102150405")
+	validatorKeyFile := directory + params.BeaconConfig().ValidatorPrivkeyFileName + t.Format("20060102150405")
```

Comment:
not sure what is the point of this `t.Format("20060102150405")`

Reply:
to create multiple validators and withdrawal keys in the same directory while not running over each other and time stamping them

Reply:
is there a better way to mark them, maybe use the pubkey as the suffix ? Having the time doesnt help in differentiating them.